pages/statistics.tsx

```diff
@@ -1,5 +1,5 @@
-import styles from "@/styles/pages/statistics.module.scss";
-import Stat from "@/components/Stat";
+import styles from "../styles/pages/statistics.module.scss";
+import Stat from "../components/Stat";
 
 const data = [
```

Comment:
give this a type - The type
```ts
{
    id: number;
    amount: number;
    name: string;
}[]
```
is inferred correctly but you want to limit what people can write in the variable declaration...
Notice that if one of the rows I write
```js
{ id: 6, amount: 1, name: "accounts", foo: false }
``` 
than the type becomes:
```ts
({
    id: number;
    amount: number;
    name: string;
    foo?: undefined;
} | {
    id: number;
    amount: number;
    name: string;
    foo: boolean;
})[]
```
which is incorrect and can lead to a trailing error

Reply:
noted